### PR TITLE
Stricter atom/New enforcement.

### PR DESCRIPTION
**Pull request by afterthought2**
The remaining New overrides are:

- mapdata obj; needed
- lighting and mimic overlays; desirable
- unit test obj; unclear
- /area; probably desirable
- /atom; needed

### CODE REVIEW

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -214,52 +214,4 @@ area/space/atmosalert()
...
-	for(var/mob/living/carbon/human/H in src)
-		if(H.client)
-			mysound.status = SOUND_UPDATE
-			to_chat(H, mysound)
-			if(S)
-				spawn(sound_delay)
-					sound_to(H, S)
-
-	spawn(60) .()
+	forced_ambience = list('sound/ambience/seag1.ogg','sound/ambience/seag2.ogg','sound/ambience/seag3.ogg')
```
Comment (PsiOmegaDelta) [inline code/game/area/Space Station 13 areas.dm]: I'll merely note that the (forced) ambiance system will pick one ambiance and play it on repeat forever, unlike the replaced one that selects a new sound every 6 seconds (not considering the 25% probability-thing)

Comment (PsiOmegaDelta) [reply code/game/area/Space Station 13 areas.dm]: Having listened to these on endless repeat with no pause between, just no.

Comment (afterthought2) [reply code/game/area/Space Station 13 areas.dm]: Back to old system.

**Review by PsiOmegaDelta — APPROVED** [review]

File: test/check-paths.sh [hunk 2]
```diff
@@ -37,8 +37,7 @@ exactly 26 "text2path uses" 'text2path'
 exactly 3 "update_icon() override" '/update_icon\((.*)\)'  -P
 exactly 1 "goto uses" 'goto '
 exactly 468 "spawn uses" 'spawn\s*\(\s*(-\s*)?\d*\s*\)' -P
-exactly 3 "obj/New uses" '^/obj.*/New\('
-exactly 0 "relatively pathed /New calls" '\tNew\('
```
Comment (Lohikar) [inline test/check-paths.sh:41]: As said on discord, pls use `^\s+([A-z0-9_]+/)*New\(.*\).*$` with an exactly of 1 instead of just deleting this.

Comment (Lohikar) [reply test/check-paths.sh:41]: Whoop, bash is interpreting that `$` as a var or something, use:
`^\s+([A-z0-9_]+/)*New\(.*\).*\$`

Comment (afterthought2) [reply test/check-paths.sh:41]: Above don't work with the bash script in the unit test environment or locally on wsl/ubuntu/bash. Can add in a separate PR if you have a working alternative.

**Pull request merged**